src/components/IOUConfirmationList.js

```diff
@@ -208,7 +214,7 @@ class IOUConfirmationList extends Component {
                         forceTextUnreadStyle
                         canSelectMultipleOptions={this.props.hasMultipleParticipants}
                         disableFocusOptions
-                        selectedOptions={this.props.hasMultipleParticipants && this.getAllOptionsAsSelected()}
+                        selectedOptions={this.props.hasMultipleParticipants ? this.getAllOptionsAsSelected() : []}
```

Comment:
This fixes a propType warning (because when there are not multiple participants, then a boolean is passed as the prop instead of an array like the propTypes are expecting.